src/othermbc.md

```diff
@@ -16,22 +16,23 @@ outputs, but this appears not to have been done in licensed games.
 
 ### EMS
 
-PinoBatch learned the game selection protocol for EMS flash carts from
-beware, who in turn learned it from nitro2k01. Take this with a grain of
-salt, as it hasn't been verified on the authentic EMS hardware.
+::: warning VERIFICATION NEEDED
```

Comment:
We should decide on a consisted title for this kind of boxes. What about "Speculation"?

Reply:
Or maybe follow Wikipedia and use "Verification needed"